Epsilon.Abstractions/Component/CompetenceProfile.cs

```diff
@@ -29,6 +29,6 @@ public void AddToWordDocument(MainDocumentPart mainDocumentPart)
             );
         }
 
-        mainDocumentPart.Document.AppendChild(body);
+        // mainDocumentPart.Document.AppendChild(body);
```

Comment:
Don't comment out random code without explaining why, also commented code should never be committed so please remove it